extra/release_tool.py

```diff
@@ -394,6 +394,8 @@ def filter_docker_compose_files_list(list, version):
         "docker-compose.enterprise.yml",
         "docker-compose.auditlogs.yml",
         "docker-compose.connect.yml",
+        "docker-compose.config.yml",
+        "other-components-docker.yml",
```

Comment:
Mmmm. This should be harmless even though not strictly necessary I think.

But I agree in having the file list consistent across maintained release branches. Thanks!